script/templates/DeployMorphoVault.s.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.19;
+
+import "@forge-std/Script.sol";
+import "@forge-std/Test.sol";
+import {stdJson} from "@forge-std/StdJson.sol";
+
+import "@protocol/utils/ChainIds.sol";
+import {AllChainAddresses as Addresses} from "@proposals/Addresses.sol";
+import {IMetaMorphoFactory} from "@protocol/morpho/IMetaMorphoFactory.sol";
+import {IMetaMorpho} from "@protocol/morpho/IMetaMorpho.sol";
+
+/// @notice Template script: Deploy a MetaMorpho vaultAddress from JSON config
+contract DeployMorphoVault is Script, Test {
+    using ChainIds for uint256;
+    using stdJson for string;
+
+    struct VaultConfig {
+        string addressName; // key used in Addresses registry, e.g. "llUSDC_METAMORPHO_VAULT"
+        string vaultName; // ERC4626 name
+        string vaultSymbol; // ERC4626 symbol
+        string assetName; // name key to resolve via Addresses.getAddress(), e.g. "USDC"
+        string saltString; // arbitrary string to hash into a salt, e.g. "llUSDC"
+        uint256 initialTimelock; // initial timelock duration
+        string curatorName; // optional: name key for initial curator to set
+    }
+
+    function run() external {
+        // TODO: BASE_FORK_ID.createForksAndSelect fails
+        // Setup fork for Base chain
+        BASE_FORK_ID.createForksAndSelect();
+        // vm.createSelectFork("base");
+
+        string memory configPath = vm.envString("NEW_VAULT_PATH");
+        string memory json = vm.readFile(configPath);
+
+        VaultConfig memory cfg = _parseConfig(json);
+
+        Addresses addresses = new Addresses();
+
+        // If already registered, skip
+        if (addresses.isAddressSet(cfg.addressName)) {
+            console.log("Vault already exists for name:", cfg.addressName);
+            console.log("Address:", addresses.getAddress(cfg.addressName));
+            return;
+        }
```

Comment:
if the vault already exist there is no reason to run this script, lets revert then